frappe/social/doctype/energy_point_log/energy_point_log.py

```diff
@@ -318,7 +318,10 @@ def send_summary(timespan):
 
 	from_date = getdate(from_date)
 	to_date = getdate()
-	all_users = [user.email for user in get_enabled_system_users()]
+
+	# select only those users that have energy point email notifications enabled
+	all_users = [user.email for user in get_enabled_system_users() if
```

Comment:
What happens when you try to `sendmail` if the `all_users` key is empty?

Reply:
`sendmail` calls `send` which has a condition that if `recipients` (which is what `all_users` becomes) is none, it will return